### ISSUE TITLE
Fingerprint generator: does not generate iOS fingerprint

### ISSUE THREAD

**Issue by rviar** [apify/fingerprint-suite#92]
**Describe the bug**
I have error:
```
Uncaught Error: No headers based on this input can be generated. Please relax or change some of the requirements you specified.
```

when try to generate fingerprint with **mobile+ios+safari** input, I'm also try only **mobile+ios** input and **mobile+safari**
Always have an error;

**To Reproduce**
```javascript
const fpGenerator = new FingerprintGenerator();

const fingerprint = fpGenerator.getFingerprint({
  devices: ['mobile'],
  browsers: ['safari'],
  operatingSystems: ['ios'],
});
```

**Expected behavior**
Correct fingerprint instead Error

**System information:**
- Lib version: 2.1.1
- OS: MacOS 13.0 (arm64)
- Node.js version v16.17.1

Is there any solution?

**Comment by B4nan** [comment]
Just because you didn't get a response within 48 hours does not mean the project is dead. 

I'd expect this is an issue with the data we are gathering, not so much a bug in code - this is a validation telling you something is not possible (currently).

@barjin can you please verify?

**Comment by barjin** [comment]
Thank you for submitting this issue @rviar! (...and sorry for the little wait). Turns out, you unearthed a flaw in our generative networks generation - thank you very much! The problem was hidden in the screen size filtering we've recently added.

Fixed in `fingerprint-generator@2.1.2` (more like `header-generator@2.1.2`). Added tests so this doesn't happen again.
